app/views/users/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>User Detail</h1>
-<h2>Posts</h2>
+<div class="container p-2">
+  <div class="row flex flex-column">
+    <%= render "shared/user_header", user: @user%>
+    <div class="bio border p-3 w-50">
+      <h4>Bio</h4>
+      <p> <%= @user.bio%> </p>
+    </div>
+    <div class="w-50 border p-3 mt-1">
+      <% @user.posts.each do |post| %>
+        <%= render "shared/post_comment", post: post %>
+      <% end %>
+    </div>
+    <%= link_to "See All Post", user_posts_path(@user), class:"btn btn-secondary w-50 mt-2" %>
+  </div>
```

Comment:
- [ ] Kindly center the elements as well to match [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACBA7J7KGYMVA3EGE46DPJMEZBN2DTQ) and make them wider.

  ![image](https://user-images.githubusercontent.com/81468675/230176877-290b4ddd-520d-4708-960c-ddc95b0c72ca.png)